src/pvsite_forecast.py

```diff
@@ -130,7 +179,7 @@ def pvsite_forecast_page():
 
         # get site from database, if india set day_ahead_timezone_delta_hours to 5.5 hours
         with connection.get_session() as session:
-            site = get_site_by_uuid(session, site_selection_uuid)
+            site = site_uuids(session, site_selection_uuid)
```

Comment:
im not sure why this has changed? please change this back

Reply:
reverted back to original